examples/chart/teleport/templates/deployment.yaml

```diff
@@ -17,14 +17,26 @@ spec:
       serviceAccount: {{ .Chart.Name }}
       containers:
       - name: teleport
-        image: quay.io/gravitational/teleport-ent:{{ .Chart.Version }}
-        imagePullPolicy: Always
-        resources:
-          requests:
-            cpu: 1m        
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+        - --insecure-no-tls
+        # See https://gravitational.com/teleport/docs/admin-guide/#ports
         ports:
-        - containerPort: 8080
-          name: keygen
+        - containerPort: 3022
+          name: node-ssh
+        - containerPort: 3023
+          name: proxy-ssh
+        - containerPort: 3024
+          name: proxy-tunnel
+        - containerPort: 3025
+          name: auth-ssh
+        - containerPort: 3080
+          name: proxy-web
+        - containerPort: 3026
+          name: proxy-kube
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
```

Comment:
You should never hard-code, or even set resource requests/limits. Make it completely up to the users, and use the idiomatic key `resources` in `values.yaml` for that.